about.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Us</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="page-container">
+      <div class="content-wrap">
+        <div class="topbar">
+          <a class="text-decoration-none me-4" href="">
+            <i class="fa-brands fa-facebook"></i>
+          </a>
+          <a class="text-decoration-none me-4" href="">
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+          <a class="text-decoration-none me-4" href="">Log Out</a>
+          <a class="text-decoration-none me-4" href="">My Page</a>
+          <a class="text-decoration-none me-4" href="">Burmese</a>
+        </div>
+        <section class="wrap pb-5">
+          <header class="menu-wrap">
+            <nav>
+              <input
+                type="checkbox"
+                class="toggler"
+                id="toggle"
+                onclick="toggle()"
+              />
+              <div class="hamburger"><div></div></div>
+              <div class="menu">
+                <div class="overlay">
+                  <div>
+                    <ul class="menubar">
+                      <li>
+                        <a href="index.html" class="brand">
+                          <img src="./img/logo.png" alt="logo" />
+                        </a>
+                      </li>
+                      <li>
+                        <ul class="nav-wrap">
+                          <li>
+                            <a href="about.html" class="nav-links">About</a>
+                          </li>
+                          <li><a href="#" class="nav-links">Program</a></li>
+                          <li>
+                            <a href="#works" class="nav-links">Buy Tickets</a>
+                          </li>
+                          <li>
+                            <a href="#works" class="nav-links">Sponsor</a>
+                          </li>
+                          <li><a href="#works" class="nav-links">News</a></li>
```

Comment:
- [ ] It would be nice if we would add a link to navigate us to the main page on the about page